### DISCUSSION TITLE
When does Silent have Progress?

### DISCUSSION THREAD

**Discussion by aaronliu0130** [microsoft/winget-pkgs#118849] [Q&A]
How do I determine if a package's silent install should be SilentWithProgress or just Silent?

**Comment by mdanish-kh** [comment]
You need to check whether the installer shows any windows or progress relation information during installation.

Take [Oh my posh's](https://github.com/microsoft/winget-pkgs/blob/bdbe60cfbf3e9bde8caf2824350d0fa914cc4af1/manifests/j/JanDeDobbeleer/OhMyPosh/18.6.3/JanDeDobbeleer.OhMyPosh.installer.yaml#L23) installer as an example.

If you pass in `./installer.exe /SILENT`, you will see that the installer shows progress during installation, making `/SILENT` a SilentWithProgress switch and if you pass in `./installer.exe /VERYSILENT` instead, it shows no windows or progress, making `/VERYSILENT` a Silent switch

**Reply by aaronliu0130** [reply]
Hmm. I have this nullsoft installer (#118844) that immediately starts installing showing the progress bar without any user input and closes after finishing. However the Azure Pipelines run couldn't finish. If I pass the nullsoft /S it'll just give no progress at all.
